runtime/queries/rust/highlights.scm

```diff
@@ -143,24 +151,28 @@
   "mod"
   "extern"
 
-  "struct"
-  "enum"
   "impl"
   "where"
   "trait"
   "for"
 
-  "type"
-  "union"
   "unsafe"
   "default"
   "macro_rules!"
 
-  "let"
-
   "async"
 ] @keyword
 
+[
+  "struct"
+  "enum"
+  "union"
+
+  "type"
+] @keyword.storage.type
+
+"let" @keyword.local
```

Comment:
`keyword.local` doesn't seem to be a valid scope

Reply:
It exists as a consequence of `keyword.local.import` in `ledger`, but yes that was not correct, moved to `keyword.storage`